src/Security/Member.php

```diff
@@ -1343,19 +1336,17 @@ public function getLocale()
     }
 
     /**
-     * Override the default getter for TimeFormat so the
-     * default format for the user's locale is used
-     * if the user has not defined their own.
+     * Return the time format based on the user's chosen locale,
+     * falling back to the default format defined by the {@link i18n.get_locale()} setting.
      *
      * @return string ISO date format
      */
     public function getTimeFormat()
     {
-        $timeFormat = $this->getField('TimeFormat');
-        if ($timeFormat) {
-            return $timeFormat;
-        }
-        return $this->getDefaultTimeFormat();
+        $format = $this->getDefaultTimeFormat();
```

Comment:
As above, we may as well merge these. The other place getDefault* methods were called are gone.